### PR TITLE
Add support for using environment variables

### PR DESCRIPTION
**Pull request by hwdsl2**
Please review the changes - I updated my develop branch with your proposed changes to support environment variables. Also, I made the public/private IP variable names longer to avoid confusion. Now can define `$VPNSETUP_PRESET_PUBLIC_IP` and `$VPNSETUP_PRESET_PRIVATE_IP` if needed.

Let me know what you think. If these look good feel free to update your pull request, so that I can merge the changes back into my master branch.

### CODE REVIEW

File: vpnsetup.sh [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
 # Define your own values for these variables
-# - IPsec Pre-Shared Key, VPN Username and Password
 # - All values MUST be quoted using 'single quotes'
 # - DO NOT use these characters inside values:  \ " '
+# - IPsec Pre-Shared Key, VPN Username and Password
 
-IPSEC_PSK=''
-VPN_USER=''
-VPN_PASSWORD=''
+VPN_IPSEC_PSK=$VPN_IPSEC_PSK
```
Comment (dsoprea) [inline vpnsetup.sh]: I totally agree with wanting these to match.

File: vpnsetup.sh [hunk 2]
```diff
@@ -105,11 +105,14 @@ echo 'In case the script hangs here for more than a few minutes,'
...
 echo
 
-# In Amazon EC2, these two variables will be retrieved from metadata.
-# For all other servers, replace them with actual IPs or comment out.
-# If your server only has a public IP, put that IP on both lines.
-PUBLIC_IP=$(wget --retry-connrefused -t 3 -T 15 -qO- 'http://169.254.169.254/latest/meta-data/public-ipv4')
-PRIVATE_IP=$(wget --retry-connrefused -t 3 -T 15 -qO- 'http://169.254.169.254/latest/meta-data/local-ipv4')
+# In case auto IP discovery fails, you may manually enter server IPs here.
+# If your server only has a public IP, put that public IP on both lines.
+PUBLIC_IP=$VPNSETUP_PRESET_PUBLIC_IP
```
Comment (dsoprea) [inline vpnsetup.sh]: Seems redundant to say preset since all environment variables could be called this, but it's your choice. No big deal.

Comment (hwdsl2) [inline vpnsetup.sh]: Thank you, you may omit the PRESET part and just name them `VPNSETUP_PUBLIC_IP` and `VPNSETUP_PRIVATE_IP`.

**Comment by dsoprea** [comment]
I shouldn't be the one to merge this because it will conflict with the changes that I've already pushed. Go ahead and respond to the comments, and then merge if no further discussion is necessary.

**Comment by dsoprea** [comment]
I'm going to close this PR so that it doesn't hang open if there's not going to be a merge.

File: vpnsetup.sh (hunk 2 above)
Comment (dsoprea) [inline vpnsetup.sh]: Okay. Let me make the changes locally and submit (the original route we
were taking before you submitted a PR to me).

**Pull request closed without merging**